src/zepben/eas/client/eas_client.py

```diff
@@ -262,7 +262,22 @@ async def async_run_hosting_capacity_work_package(self, work_package: WorkPackag
                                 "calculatePerformanceMetrics": work_package.results_config.metrics_config.calculate_performance_metrics if work_package.results_config.metrics_config.calculate_performance_metrics is not None else None
                             } if work_package.results_config.metrics_config is not None else None
                         } if work_package.results_config is not None else None,
-                        "qualityAssuranceProcessing": work_package.quality_assurance_processing if work_package.quality_assurance_processing is not None else None
+                        "qualityAssuranceProcessing": work_package.quality_assurance_processing if work_package.quality_assurance_processing is not None else None,
+                        "generatorConfig": {
```

Comment:
There's no need for these fields to be moved to a separate generatorConfig - this change really should have just added another flag to the ModelConfig for the moment.

As an aside - we probably want to consider EAS just receiving an arbitrary JSON object of all this config that it passes on directly to the HC service, which unpacks and verifies it. This would save us having to update EAS every time we want to add a config property.